satnogsclient/web/app.py

```diff
@@ -66,23 +66,28 @@ def get_control_rx():
         tmp = {}
         tmp['log_message'] = e
         return jsonify(tmp)
-    packet_list = json.loads(data)
+    print "einai auto paketo?", data
```

Comment:
I know this is WIP, but leaving this comment to make sure that these debug messages won't be part of master log. These should be eventually removed and the commits squashed before merge.